velox/exec/tests/AggregateFunctionRegistryTest.cpp

```diff
@@ -196,4 +196,13 @@ TEST_F(AggregateFunctionRegistryTest, multipleNames) {
       ARRAY(BIGINT()));
 }
 
+TEST_F(AggregateFunctionRegistryTest, getAggregateFunctionNames) {
+  auto functions = getAggregateFunctionNames();
+  EXPECT_EQ(functions.size(), 2);
+  std::sort(functions.begin(), functions.end());
```

Comment:
No need to sort. Use EXPECT_THAT with UnorderedElementsAre: https://google.github.io/googletest/reference/matchers.html